### ISSUE TITLE
Contributor Mandatory fields not being marked as mandatory

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#745]
Attempted changing an existing Contributor's name, but got "unable to save" error message (emailed image of this) related to requiring values of email and phone number - need to mark these as mandatory?

**Comment by bferguso** [comment]
There appears to be a bug in Arches Core that is causing this. There was a PR created to fix this (https://github.com/archesproject/arches/pull/9547) but the change doesn't seem to be in the release we're using. I've reached out to the Arches team to see what the status of this is.

A patch was created for this however it was lost in the Arches Core source code. Brought it to the team's attention and they'll be merging the fix back into the 7.5.x stream. We can cherry pick that change to pick it up when it's been resolved. New issue is here for tracking: https://github.com/archesproject/arches/issues/10609

**Comment by bferguso** [comment]
The arches core fix was rolled into release 7.5.2. Marking this as complete contingent on the update to 7.5.2

**Comment by bferguso** [comment]
Automated cleanup
